packages/rocketchat-livechat/server/hooks/sendToCRM.js

```diff
@@ -49,6 +58,31 @@ RocketChat.callbacks.add('livechat.saveInfo', (room) => {
 	return sendToCRM('LivechatEdit', room);
 }, RocketChat.callbacks.priority.MEDIUM, 'livechat-send-crm-save-info');
 
+RocketChat.callbacks.add('afterSaveMessage', function(message, room) {
+	// only call webhook if it is a livechat room
+	if (!(typeof room.t !== 'undefined' && room.t === 'l' && room.v && room.v.token)) {
+		return message;
+	}
```

Comment:
This will pass if `room.t === undefined`, I'd suggest this:
```javascript
if (room.t !== 'l' || room.v == null || room.v.token == null)) {
```

Reply:
@rodrigok, fix was done according to suggestion.
This code is a copy from [sendToFacebook.js](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-livechat/server/hooks/sendToFacebook.js#L14) and [sendMessageBySMS.js](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-livechat/server/sendMessageBySMS.js#L14), maybe these callbacks need the same changes.